src/extract-metadata.js

```diff
@@ -120,13 +119,20 @@ function applyMetaRule(target, obj) {
   });
 }
 
+function globToRegExp(glob) {
+  let reString = glob.replace(/\*\*/g, '_');
+  reString = reString.replace(/\*/g, '[0-9a-z-]*');
+  reString = reString.replace(/_/g, '.*');
+  return new RegExp(`^${reString}`);
+}
+
 function filterGlobalMetadata(metaRules, url) {
   const metaConfig = {};
   metaRules.forEach((rule) => {
     const glob = rule.url || rule.URL || rule.Url;
-    if (glob && typeof glob === 'string') {
+    if (glob && typeof glob === 'string' && /[0-9a-z-/*]/.test(glob)) {
```

Comment:
are you sure about this additional test?

Reply:
Yes, we want to make sure nothing funny will bleed through to the regexp.